CONTRIBUTING.md

```diff
@@ -4,57 +4,21 @@ This project is work of [many developers](https://github.com/artsy/force/graphs/
 
 We accept [pull requests](https://github.com/artsy/force/pulls), and you may [propose features and discuss issues](https://github.com/artsy/force/issues).
 
-## Quick Start
-
-- Install dependencies:
-
-        $ git clone git@github.com:artsy/force.git
-        $ cd force
-        $ cp .env.oss .env
-        $ yarn install
-
-- To run the full Artsy.net server:
-
-        $ yarn start
-        $ open http://localhost:5000
-
-- Run the tests:
-
-        $ yarn type-check
-        $ yarn jest
-
 ## Setup Instructions
 
-### Clone the Project
-
 Clone the [project on GitHub](https://github.com/artsy/force) and cd in:
 
 ```sh
-git clone git@github.com:artsy/force.git
-cd force
+git clone git@github.com:artsy/force.git && cd force
 ```
 
-## Run Force
-
-Install [NVM](https://github.com/creationix/nvm) and Node 12:
+Execute the setup script:
 
 ```sh
-nvm install 12
-nvm alias default 12
+./scripts/setup.sh
```

Comment:
Based on our experimentation, I thought `source ./scripts/setup.sh` was the way to ensure the `nvm` choice persisted.

Reply:
I chose not to add that and instead add a "_note to nvm users_" because I was experiencing terminal window exit when script threw an error. I'll test this further and experiment with `|| return` that you pointed out so as to add `source` prefix.

Reply:
This works for me.